docker-compose.yml

```diff
@@ -94,9 +94,9 @@ services:
       - ~/.ivy2:/root/.ivy2:cached
       - ~/.m2:/root/.m2:cached
       - ~/.sbt:/root/.sbt:cached
-      - ~/.cache/coursier:/root/.cache/coursier:cached
+      - ~/.cache/coursier:/home/sbtuser/.cache/coursier:cached
     environment:
-      - SBT_OPTS=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005 -Dfile.encoding=UTF-8 -Xms512m -Xmx1536m -Xss2m -XX:ReservedCodeCacheSize=256m -XX:+TieredCompilation -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC
+      - SBT_OPTS=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005 -Dfile.encoding=UTF-8 -Xms512m -Xmx10g -Xss2m -XX:ReservedCodeCacheSize=512m -XX:+TieredCompilation -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC
```

Comment:
Xmx10g is too extreme, I would put it at 4g maximum

Reply:
Whoa. Yeah, `-Xmx10g` is waaay too high. The Github Actions CI nodes only have ~8GB ram. We need to be much more conservative here.

Reply:
@voropaevp Can you actually confirm that these changes (including `-Xmx10g`) is actually needed? I have a suspicion that you left these changes in there before I bumped the sbt version which fixed these memory related issues on m system and hence I don't think any of these changes apart from the `cache` are needed